extensions/markdown-language-features/src/languageFeatures/pathCompletions.ts

```diff
@@ -9,7 +9,7 @@ import { MarkdownEngine } from '../markdownEngine';
 import { TableOfContents } from '../tableOfContents';
 import { resolveUriToMarkdownFile } from '../util/openDocumentLink';
 import { SkinnyTextDocument } from '../workspaceContents';
-import { MdLinkProvider } from './documentLinkProvider';
+import { findCode, MdLinkProvider } from './documentLinkProvider';
```

Comment:
`findCode` is an implementation detail of  document links and should not be exported. Instead you can likely just want to have a public api version of `getLinkDefinitions` that computes the code ranges for you